Firebase/InAppMessagingDisplay/Modal/FIDModalViewController.m

```diff
@@ -123,7 +126,6 @@ - (void)viewDidLoad {
   }
 
   self.messageCardView.backgroundColor = self.modalDisplayMessage.displayBackgroundColor;
-  self.messageCardView.layer.cornerRadius = 4;
```

Comment:
Is this intentional?

Reply:
Ahh, I see it was set in the Storyboard now.

Reply:
Yup, our old favorite `IBDesignable`.